### PR TITLE
DM-11877: document recipes

### PR DESCRIPTION
**Pull request by PaulPrice**

### CODE REVIEW

**Comment by n8pease** [comment]
I think `add_<datasetType>` functions are new in cameraMapper with the recipe change? Those should be documented in this LDM somewhere.
...BTW re. the comment in code, `# Add support for configuring FITS compression`
how does the word "add" correspond to fits compression settings?

File: index.rst [hunk 1]
```diff
@@ -770,6 +770,166 @@ calibrations from separate directories: calibrations shall be read from the
...
 
+Recipes
+^^^^^^^
+
+Recipes are sets of parameters used to control the operation of a
+`Formatter`. The recipe is provided to the formatter as an
+``lsst.daf.base.PropertySet``, known as the ``additionalData``. Recipes are
+grouped by the relevant storage type and each recipe has a symbolic name which
+may be used to refer to the appropriate set of parameters. Recipes are defined
+in YAML format in ``obs_base/policy/writeRecipes.yaml``; obs packages may
```
Comment (ktlim) [inline index.rst]: Maybe say that `readRecipes` are not expected to be needed but could be added later (hence the name of this file)?

File: index.rst [hunk 2]
```diff
@@ -770,6 +770,166 @@ calibrations from separate directories: calibrations shall be read from the
...
+Recipes used for ``FitsStorage`` datasets define how the FITS images will be
+compressed. Separate sets are required for the ``image``, ``mask`` and
+``variance`` planes.
+
+Parameters are separated according to whether they are for compression or
+scaling. Scaling (a.k.a. quantization) is used to convert a floating-point
+image to an integer image, which allows higher compression factors at the
+expense of the loss of some information (typically unimportant information).
+
+The compression options are used exclusively by cfitsio. They are:
```
Comment (ktlim) [inline index.rst]: Add a reference for cfitsio.

**Review by ktlim — APPROVED** [review]
Per discussion this morning, might be good to have the new text be one-sentence-per-line.

**Comment by PaulPrice** [comment]
The `add_<datasetType>` mechanism was existing, it's just that they weren't used in obs_base until DM-11332. I believe the "add" refers to adding to the `additionalData`.

**Comment by PaulPrice** [comment]
If you don't mind, @ktlim, I'm going to leave the format as it is --- it is formatted the same as the rest of the file, and in that case I don't think there's any value to putting in the effort to change it until the rest of the file changes.

**Pull request merged**